test/helpers.bash

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Root directory of integration tests.
+INTEGRATION_ROOT=$(dirname "$(readlink -f "$BASH_SOURCE")")
+
+# Test data path.
+TESTDATA="${INTEGRATION_ROOT}/../testdata"
+
+# Root directory of the repository.
+OCID_ROOT=${OCID_ROOT:-$(cd "$INTEGRATION_ROOT/../.."; pwd -P)}
+
+# Path of the ocid binary.
+OCID_BINARY=${OCID_BINARY:-${OCID_ROOT}/ocid}
+# Path of the ocic binary.
+OCIC_BINARY=${OCIC_BINARY:-${OCID_ROOT}/ocid}
+# Path of the runc binary.
+RUNC_BINARY=${RUNC_BINARY:-${OCID_ROOT}/runc}
+
+PATH=$PATH:$BATS_TMPDIR
+
+# Run ocid using the binary specified by $OCID_BINARY.
+# This must ONLY be run on engines created with `start_ocid`.
+function ocid() {
+	sudo "$OCID_BINARY" "$@"
```

Comment:
>  a Dockerfile will also let us skip the runc build - so marking this WIP for now.

since this is WIP this whole PR was just a show off of what I'm doing - we'll drop that sudo once we're inside a docker container probably. Not sure about any plan to have ocid unprivileged.